v2/internal/controllers/crd_machinelearningservices_test.go

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package controllers_test
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-service-operator/v2/api/compute/v1beta20201201"
+	"github.com/Azure/azure-service-operator/v2/api/keyvault/v1beta20210401preview"
+	machinelearningservices "github.com/Azure/azure-service-operator/v2/api/machinelearningservices/v1beta20210701"
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1beta20201101"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1beta20200601"
+	storage "github.com/Azure/azure-service-operator/v2/api/storage/v1beta20210401"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+	"github.com/Azure/go-autorest/autorest/to"
+	v1 "k8s.io/api/core/v1"
+)
+
+func Test_MLS_Workspaces_CRUD(t *testing.T) {
```

Comment:
minor: IMO name this `Test_MachineLearning_Workspaces_CRUD`. It's clearer than `MLS`. Applies for below too